setup.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+from setuptools import setup
+
+import typeshed
+
+
+if typeshed.__version__.endswith(".dirty"):
```

Comment:
I still don't like that there is a typeshed package whose `__init__.py` runs a bunch of subprocesses, just so that setup.py can use a dynamically generated `__version__`. (For mypy I don't mind since it's an app not a library.) Other projects I've seen just put the version number in setup.py -- if you're going to dynamically generate the version why not put the whole thing in setup.py? And why does typeshed need to be a package? Who besides setup.py would ever import it?

Reply:
Alright, I agree that subprocessing with every import of `typeshed` is spurious. I'll change that.